tests/integration/dump_test.go

```diff
@@ -761,6 +776,33 @@ func Test_Dump_Sanitize_Special_Entities(t *testing.T) {
 			stateFile: "testdata/dump/008-sanitizer/ca-cert.yaml",
 			runWhen:   func(t *testing.T) { runWhen(t, "kong", ">=2.8.0") },
 		},
+		{
+			name:           "dump sanitize env vault and vault references",
+			stateFile:      "testdata/dump/008-sanitizer/env-vault.yaml",
+			runWhen:        func(t *testing.T) { runWhen(t, "enterprise", "3.4.0") },
+			skipValidation: true, // env vault validation endpoint not available in 3.4.0
+		},
+		{
+			name:      "dump sanitize env vault and vault references",
+			stateFile: "testdata/dump/008-sanitizer/env-vault.yaml",
+			runWhen:   func(t *testing.T) { runWhen(t, "enterprise", ">=3.5.0") },
+		},
+		{
+			name:           "dump sanitize vault config",
+			stateFile:      "testdata/dump/008-sanitizer/vault-configs.yaml",
```

Comment:
Same for this and next

Reply:
Same as above. The differentiator is skipValidation param.